data/features/wsgi.yaml

```
@@ -0,0 +1,6 @@
+---
+weight: 130
```

```suggestion
weight: 41
```
Put it after deployment?